auxx/Main.hs

```diff
@@ -4,38 +4,59 @@ module Main
 
 import           Universum
 
-import           Control.Exception.Safe (handle)
-import           Data.Maybe (fromMaybe)
-import           Formatting (sformat, shown, (%))
-import           Mockable (Production (..), runProduction)
-import qualified Network.Transport.TCP as TCP (TCPAddr (..))
+import           Control.Exception.Safe
+    (handle)
+import           Data.Maybe
+    (fromMaybe)
+import           Formatting
+    (sformat, shown, (%))
+import           Mockable
+    (Production (..), runProduction)
+import qualified Network.Transport.TCP as TCP
+    (TCPAddr (..))
```

Comment:
Quite honestly, I *really* dislike that. It will take me a *long* time to come to terms with it.

And as I stated above, what does qualified and explicit imports buy us here? if that was just a qualified import:
```
import qualified Network.Transport.TCP as TCP
```
uses of the `TCPAddr` constructors still need to be explicit, eg `TCP.Unaddressable`.

Reply:
I have to say, I agree with Erik again here. I know that's just a visual thing, but it's different from 99% of project I have worked on (or 99% of the Haskell projects out there), and I think being consistent is a good thing, in general 😢

Reply:
I 100% agree that having a qualified import with explicit imports is silly.